yugabytedb_managed/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Not sure why this is complaining. I've added two svg files to `assets/logos` as specified in https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#complete-the-necessary-integration-asset-files. I've also ran the validate cmd locally without any errors returned:
```
$ ddev validate all yugabytedb_managed
Running validations for extras repo ...
---
Skipping agent-reqs
---
Executing validation codeowners
All integrations have valid codeowners.

---
Executing validation config
Validating default configuration files for 1 checks...
yugabytedb_managed:
Did not find spec file /Users/daniel/dd/integrations-extras/yugabytedb_managed/ for check yugabytedb_managed

---
Executing validation dashboards
Validating Dashboard definition files for 1 checks...
1 valid files

---
Skipping dep
---
Skipping eula
---
Executing validation jmx-metrics
Validating JMX metrics files for 0 checks ...
0 total JMX integrations
0 valid metrics files

---
Executing validation imports
Validating imports for 1 checks to avoid deprecated modules ...
Validation passed!

---
Executing validation manifest
Validating manifest.json files for 1 checks ...

---
Executing validation models
Validating data models for 1 checks ...

---
Executing validation package
Validating files for 0 checks ...

---
Executing validation readmes
All 1 READMEs are valid!

---
Executing validation saved-views
Validating saved views for 1 checks ...
All saved views are valid!

---
Executing validation service-checks
Validating service_checks.json files for 1 checks ...
1 valid files
```